VirtoCommerce.CoreModule.Web/Module.cs

```diff
@@ -87,9 +87,22 @@ public override void Initialize()
 
             _container.RegisterType<ISearchPhraseParser, SearchPhraseParser>();
 
+            // Allow scale out of indexation through background worker, if opted-in.
+            if (ConfigurationHelper.GetAppSettingsValue("VirtoCommerce:Search.ScaleOutIndexation", false))
```

Comment:
Need to add this setting in the module.manifest, then it will available in the manager UI.

Reply:
Check